### PR TITLE
AnonCreds Quarterly Report - 2024Q2

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+## Releases
+
+- [anoncreds-rs] -- v0.2.0
+- [anoncreds-clsignatures-rs] — v0.2.4, v0.3.0, v0.3.1, v0.3.2
+
+## Overall Activity in the Past Quarter
+
+Coding for the quarter focused enabling the use of AnonCreds v1 in W3C
+Verifiable Credential Data Model ([VCDM]) format. This was triggered by a "Code
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md:91]: Is [VCDM] supposed to be a link?

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+With Us" procurement from BC Gov that was won by [DSR Corporation] for the
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md:92]: Missing link for [DSR Corporation]

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+[AnonCreds Rust] work, [Animo Solutions] for the adding support for the format
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md:93]: Missing link for [Animo Solutions]

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+in the [Credo-TS] and [Bifold Wallet] libraries (formerly Aries Framework
+JavaScript and Aries Bifold React Native, respectively and now part of the [Open
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md]: Missing link for [Open Wallet Foundation]

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+Wallet Foundation]), and What's Cookin for adding support the for the format in
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md]:
```suggestion
Wallet Foundation]), and What's Cookin for adding support for the format in
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2024/2024-Q2-Hyperledger-AnonCreds.md:91]: Yes - fixed

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**